recipes/minikube_setup/configure.py

```
@@ -0,0 +1,148 @@
+    cmd = [
+        "helm", "install", "--wait", "--generate-name",
+        "--kube-context", "minikube",
+        "-n", "gpu-operator", "--create-namespace",
+        "nvidia/gpu-operator",
+        f"--version={GPU_OPERATOR_VERSION}",
+    ]
+    if driver_installed:
+        # Host driver is already loaded; tell the operator not to install its own
+        cmd += ["--set", "driver.enabled=false"]
+
+    check_call(cmd)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)